action.yml

```
@@ -14,6 +14,14 @@ inputs:
   anaconda_nightly_upload_token:
     description: 'Token to upload to scientific python org'
     required: true
+  anaconda_nightly_upload_url:
+    description: 'URL to upload to scientific python org'
+    required: false
+    default: scientific-python-nightly-wheels
```

```suggestion
    default: scientific-python-nightly-wheels
```

I don't know how the action works, but this is confusing since it isn't a URL.
If it can be anything other than a URL, that should either be documented or the field be renamed.

---

So the variable name is wrong! it should be `anaconda_nightly_upload_organization`